extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -405,10 +405,21 @@
 		_t.image = parentNode.querySelector("img");
 		_t.updateInterval = parseInt(parentNode.getAttribute("data-update-interval"), 10);
 
-		_t.url = _t.image.getAttribute("src").replace(/&t=\d+/g, "");
+		_t.url = parentNode.getAttribute("data-proxied-url");
+		_t.validUrl = parentNode.getAttribute("data-valid-url");
 
-		_t.setValuePrivate = function() {
-			_t.image.setAttribute("src", _t.url + "&t=" + Date.now());
+		_t.setValuePrivate = function(value, itemState) {
+			if (itemState.startsWith("data:")) {
+				// Image element associated to an item of type ImageItem
+				_t.image.setAttribute("src", itemState);
```

Comment:
Neither do many of us. Just in case we can't find anyone with Safari or you need to debug it, https://appetize.io/ — you can use iPhone on the front page for some amount of time, also there is some free time if you register an account